shared/scripts/lib/utils/tipsy-txt-parser.js

```diff
@@ -109,6 +111,15 @@ export function parseTxt() {
         serialInfo.prevTime = Date.now().toString();
         localStorage.setItem(document.domain, JSON.stringify(serialInfo));
       }
+    } if (req.status == 304) {
+      info = localStorage.getItem(document.domain);
+      if (info !== undefined || info !== '') {
```

Comment:
(I don’t get emails from this repo, so I’m not notified of the review.) You’re complaining about existing code and not my code changes, right? I’m checking to make sure there is a locally stored item (cached information) before updating the `prevTime`. All other uses of `setItem` replaces the existing item where as this updates the item already in storage. I can refactor everything here, but that would be a separate issue.